feature-utils/poly-look/test/react-components/contexts/mocks/data-importers-mock.js

```diff
@@ -0,0 +1,6 @@
+class MockImporterClass {
```

Comment:
It's a bit suspicious to call a class "class" - I was hoping Hungarian notation was gone :D